package/python-kiwi-spec-template

```diff
@@ -291,6 +291,7 @@ Provides:       kiwi-filesystem:ext3
 Provides:       kiwi-filesystem:ext4
 Provides:       kiwi-filesystem:squashfs
 Provides:       kiwi-filesystem:xfs
+Provides:       kiwi-filesystem:erofs
```

Comment:
We also need `Requires: erofs-utils`

Reply:
yes saw it a second before and added it. But probably not correct as I only added for TW, please advise

Reply:
I just did in the comment about the conditional. :)

The provides for erofs needs to be similarly wrapped because otherwise it'll be a problem.